go.mod

```diff
@@ -27,6 +27,7 @@ require (
 	github.com/spf13/cobra v1.7.0
 	github.com/spf13/viper v1.16.0
 	github.com/stretchr/testify v1.8.4
+	golang.org/x/exp v0.0.0-20220827204233-334a2380cb91
```

Comment:
I literally _just_ got rid of this lib in #367...

Reply:
removed